samples/cellular/location/sample.yaml

```diff
@@ -72,12 +72,16 @@ tests:
   sample.cellular.location.thingy91x:
     sysbuild: true
     build_only: true
+    integration_platforms:
+      - thingy91x/nrf9151/ns
```

Comment:
Location sample is handled in PR #15266. If you want, you can copy those minor changes into this PR or drop location sample from this PR.

Reply:
I'll drop my changes here.